client/js/components/content-slider.js

```diff
@@ -305,32 +306,29 @@ const contentSlider = (el, options) => {
   function nextSlide(e) {
     if (e.target.classList.contains(classes.sliderControlInactive)) return;
     const moveToPosition = positionIndex + 1;
-    trackEvent(getTrackingEvent('next', {moveToPosition}));
+    trackGaEvent({
+      category: 'component',
+      action: `content-slider-button:${e.type}`,
+      label: `id:${id}, type:next, to-position:${moveToPosition + 1}, total-items:${sliderElements.slideImages.length}`
+    });
     return updatePosition(moveToPosition, positionArray);
   }
 
   function prevSlide(e) {
     if (e.target.classList.contains(classes.sliderControlInactive)) return;
     const moveToPosition = positionIndex - 1;
-    trackEvent(getTrackingEvent('next', {moveToPosition}));
```

Comment:
Explains why we weren't seeing any 'prev' button clicks in Mixpanel

Reply:
Ah, good spot